mreg/models.py

```diff
@@ -289,7 +289,7 @@ def get_ipaddresses(self):
 
         def _add_to_result(ip, ttl, hostname):
             # Wildcards are not allowed in reverse zones.
-            if "*" in hostname:
+            if "*" in hostname:  # pragma: no cover
```

Comment:
Remove pragma here. Instead extend tests.

Reply:
I was trying to find a nice way to test this in the reverse zone tests, without success. Suggestions are welcome. :)